docs/installation.rst

```diff
@@ -14,26 +14,38 @@ Installing ``openfe``
 When you install ``openfe`` through any of the methods described below, you
 will install both the core library and the command line interface (CLI). 
 
-Installation with ``conda`` (recommended)
+Installation with ``micromamba`` (recommended)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-We recommend installing ``openfe`` with ``conda``, because it provides easy
+We recommend installing ``openfe`` with ```micromamba`` <https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html#micromamba>`_ , because it provides easy
 installation of other tools, including molecular dynamics tools such as
-OpenMM and ambertools, which are needed by ``openfe``. 
-Conda can be installed either with the `full Anaconda distribution
-<https://www.anaconda.com/products/individual>`_, or with
-the `smaller-footprint miniconda
-<https://docs.conda.io/en/latest/miniconda.html>`_.
+OpenMM and ambertools, which are needed by ``openfe``.
 
-You can get ``openfe`` from the ``conda-forge`` channel. To install the most
-recent release in its own environment, use the commands ::
+To install ``micromamba`` execute the `installation script <https://mamba.readthedocs.io/en/latest/installation.html#install-script>`_.
```

Comment:
Maybe a sentence here explaining that micromamba is conda on steroids.  People might have heard of conda and think it's what they should be using.

Reply:
Done!